src/computation/logic/utils.ts

```diff
@@ -281,3 +282,24 @@ export function distributeLevelingSeatsOnDistrictsPre2005(
     }
     return partyRestQuotients;
 }
+
+/**
+ * Breaks ties in the distribution of items on names
+ *
+ * @param winners The list of multiple winners from the distribution stage
+ * @param baseValue The dictionary from winners to their respective numerators
+ */
+export function breakTies(winners: KeyValuePair[], baseValue: Dictionary<number>): KeyValuePair {
```

Comment:
What is this KeyValuePair? That seems suspicious considering there are both Maps and a "custom" Dictionary in our code base already.

Reply:
Yes, I noticed it as well. It is tied to the SortedReverseDict, so I thought it would be better to make another PR to simplify the functions of it.

Reply:
Maybe make an issue out of it?

Reply:
Created issue #144